docs/blog_post.md

```diff
@@ -0,0 +1,114 @@
+# **gem5 GUI**
+
+
```

Comment:
Can you add a splash "image" which is a gif example of the GUI working? I think that will be a cool "grab".